src/js/background/assignManager.js

```diff
@@ -245,9 +240,48 @@ const assignManager = {
         delete this.canceledRequests[options.tabId];
       }
     },{urls: ["<all_urls>"], types: ["main_frame"]});
+    this.getPermissions();
+  },
 
+  getPermissions() {
+    browser.permissions.getAll().
```

Comment:
Can we use `async`/`await` here instead of `then()`?
```js
async getPermissions() {
  const {permissions} = await browser.permissions.getAll();
  permissions.includes("bookmarks") ? this.makeBookmarksMenu() : browser.contextMenus.remove(this.OPEN_IN_CONTAINER);
}
```